tests/test-data/gen-test-signatures.sh

```diff
@@ -0,0 +1,78 @@
+#!/bin/bash
```

Comment:
I skipped reviewing this file for now. I guess I'll probably review it mostly indirectly by looking at what it outputs and how the tests use the outputs.